core/src/test/java/org/elasticsearch/ingest/IngestClientIT.java

```diff
@@ -60,28 +59,6 @@
         return pluginList(IngestPlugin.class);
```

Comment:
This test tests the Java client integration, and the rest tests have similar tests as well. But maybe we should use `ESSmokeClientTestCase` here instead?